temporal-spring-boot-autoconfigure-alpha/src/main/java/io/temporal/spring/boot/autoconfigure/template/ServiceStubsTemplate.java

```diff
@@ -113,7 +113,8 @@ private void configureMTLS(
       @Nullable ConnectionProperties.MTLSProperties mtlsProperties,
       WorkflowServiceStubsOptions.Builder stubsOptionsBuilder) {
     if (mtlsProperties == null
-        || (mtlsProperties.getKeyFile() == null && mtlsProperties.getCertChainFile() == null)) {
+        || (mtlsProperties.getKeyFile() == null && mtlsProperties.getCertChainFile() == null)
```

Comment:
I don't think this is the right condition. Think about the following: it's totally fine to have only keyFile and chainFile specified. But we will `return` here.

I think you need to && all of them. keyFile, certChainFile, key, certChain.

Reply:
you're absolutely right! fixed in https://github.com/temporalio/sdk-java/pull/1409/commits/7e4269de3834c108c535bc753d15e5f46263fbb1